docs/source/quark_script.rst

```diff
@@ -1421,3 +1433,48 @@ Quark Script Result
    $　python3 CWE-295.py
    Requested API level 29 is larger than maximum we have, returning API level 28 instead.
    CWE-295 is detected in method, Lcom/insecureshop/util/CustomWebViewClient; onReceivedSslError (Landroid/webkit/WebView; Landroid/webkit/SslErrorHandler; Landroid/net/http/SslError;)V
+
+
+Detect CWE-489 in Android Application (allsafe.apk, AndroGoat.apk, pivaa.apk)
+-------------------------------------------------------------------------------
+
+This scenario seeks to find **active debug code** in the APK file. See `CWE-489 <https://cwe.mitre.org/data/definitions/489.html>`_ for more details.
+
+Let's use `allsafe.apk <https://github.com/t0thkr1s/allsafe>`_, `AndroGoat.apk <https://github.com/satishpatnayak/AndroGoat>`_, `pivaa.apk <https://github.com/HTBridge/pivaa>`_ and the above APIs to show how Quark script find this vulnerability.
+
+First, we use Quark API ``getApplication`` to get the application data in the manifest. Then we use ``applicationInstance.isDebuggable`` to check if the application sets the attribute ``android:debuggable=true``. If **Yes**, that causes CWE-489 vulnerabilities.
+
+Quark Script CWE-489.py
+===========================
+
+.. code-block:: python
+
+    from quark.script import getApplication
+
+    SAMPLE_PATH = "allsafe.apk"
+
+    if getApplication(SAMPLE_PATH).isDebuggable():
+        print(f"CWE-489 is detected in the {SAMPLE_PATH}")    
```

Comment:
The sentence is not grammatically correct. The article, the, should not be put in front of a proper noun.